lib/trento_web/controllers/v1/host_controller.ex

```diff
@@ -191,15 +191,11 @@ defmodule TrentoWeb.V1.HostController do
     %{solution: solution} = OpenApiSpex.body_params(conn)
     %{id: host_id} = host
 
-    Logger.info(
-      "Operation authorized. Send real saptune_solution_apply with #{solution} to #{host_id}"
-    )
-
-    # Request operation, get operation id when created and return in json
-
-    conn
-    |> put_status(:accepted)
-    |> json(%{operation_id: UUID.uuid4()})
```

Comment:
At the end I'm removing the `operation_id` value from the response. We are not using it by now.

Reply:
Forget about it. We are going to return the `operation_id`. It will be used at least by the activity logging